### ISSUE TITLE
Mobile OSes: Localization

### ISSUE THREAD

**Issue by lucasmz-dev** [eylenburg/eylenburg.github.io#53]
Currently, not all of OSes here support localization, it would be nice to have a column for this, especially if one is trying to give someone who doesn't know English, an older person, a phone with a private OS. Calyx does, Lineage does, DivestOS apparently doesn't but most things are from Lineage apparently, so it's mostly there except for a few things, but their own apps are also translatable. I'm not sure about the other OSes in the list.

**Comment by eylenburg** [comment]
> With every OS here being based on AOSP, 99% of the OS is translated to many different languages by default. The only thing that requires translation is OS-specific strings added etc.

That's true. I don't think it's worth adding a row for this.

**Comment by lucasmz-dev** [comment]
99% is an unrealistic number. Sometimes not even AOSP is translated properly or fully. The impact that untranslated strings cause is not the same as the sum of strings that are or not translated; 

On the crowdsourced comment, that's valid, things do happen in crowdsourced translations and as someone managing both CalyxOS and LineageOS translations for PT-BR, I have seen that a few times. But 1. There are multiple ways of doing crowdsourced translations, like having proofreaders in the case of LineageOS and 2. Even professional, paid for translators in a closed system make mistakes! I notice this on AOSP strings sometimes, and it's even more annoying because it's not like someone who notices this and is impacted by this can always contact the people in charge of it easily. 

Localization is accessibility, and so it is important. It's easy to ignore this if you only know English. 

AOSP is also merely a base, most things in the OS are usually replaced with proper apps, options, etc. These don't have translations that come from AOSP.

There's also the social privacy question, you want more people to have access to private tools, because if you share data with someone who doesn't use them, then that data that is yours is also at risk.

TL:DR; Not a great number, but valid point, though it can be minimized with proper tools
